qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -211,6 +211,7 @@ def _generate_circuit(
                 circuits.append(rb_circ)
         return circuits
 
+    @cached_method
```

Comment:
This can cause a bug (or breaking API change) because by default constructor `seed` is set to None which doesn't fix the random number generator. So calling circuits generates different RB circuits every time, but transpiled circuit is cached here. Same for IRB (maybe QV as well?)